airflow/models/xcom.py

```diff
@@ -723,6 +723,21 @@ def __eq__(self, other: Any) -> bool:
             return all(x == y for x, y in z)
         return NotImplemented
 
+    def __getstate__(self) -> Any:
+        # We don't want to go through the trouble of serializing the entire
```

Comment:
```suggestion
        # We don't want to go to the trouble of serializing the entire
```
sorry, i had it wrong earlier... source : https://dictionary.cambridge.org/us/dictionary/english/go-to-the-trouble-to

Reply:
Huh TIL, I’ve been saying _go through the trouble_ all my life as well!

Reply:
maybe it is through! what do i know! maybe _to_ is british

Reply:
So from a quick search the idiom is either _go to the trouble of_ or _go through with_ something, according to both Cambridge and Merriam-Webster. It’s quite conceivable people just mix those things up.

Reply:
yeah... sounds about right